Property_map/test/Property_map/CMakeLists.txt

```diff
@@ -31,6 +31,15 @@ if ( NOT Boost_FOUND )
 
 endif()
 
+find_package( OpenMesh QUIET )
+
+if ( OpenMesh_FOUND )
+  include( UseOpenMesh )
+  add_definitions( -DCGAL_USE_OPENMESH )
+else()
+  message(STATUS "Examples that use OpenMesh will not be compiled.")
```

Comment:
I guess that's not true, it will simply not be available in the compiled test.

Reply:
@sloriot  I don't understand.

Reply:
You can forget that.